src/engine/build.sh

```diff
@@ -77,6 +77,7 @@ Guess_Toolset ()
           BOOST_JAM_TOOLSET=vacpp
        fi
     elif test_uname AIX && test_path xlc; then BOOST_JAM_TOOLSET=vacpp    
+    elif test_uname FreeBSD && test_path freebsd-version; then BOOST_JAM_TOOLSET=clang
```

Comment:
>> Is this is for FreeBSD >=10.. Shouldn't the version also be checked?

`freebsd-version` is new command which is provided only if FreeBSD >= 10.

> Also, shouldn't this be testing whether clang exists?

OK, I`ll check `test_path clang`.